docs/upgrading-fcos.md

```diff
@@ -13,14 +13,125 @@ Occasionally, changes are made to Fedora CoreOS Butane configs (those that speci
 1. TOC
 {:toc}
 
-{% comment %}
-
 ## From Version 1.4.0 to Version 1.5.0
 
 There are no breaking changes between versions 1.4.0 and 1.5.0 of the `fcos` configuration specification. Any valid 1.4.0 configuration can be updated to a 1.5.0 configuration by changing the version string in the config.
 
 The following is a list of notable new features.
 
+## GRUB password
```

Comment:
Maybe `GRUB passwords`?  And this should be a level-3 heading `###` rather than level 2.